RELEASE-PLAN.md

```diff
@@ -0,0 +1,53 @@
+# Background and Context
+
+Releases of the Pathfinder Framework (Version 2 can be found [here](https://www.carbon-transparency.com/media/jpslsujn/pathfinder-framework.pdf)) are strongly
+informing the tech specs releases.
+
+Therefore, this timeline page contains the release plan for both, the Pathfinder Framework and the Pathfinder Tech Specs.
+
+In addition, the tables below disclose deprecation of releases and subsequent sunset dates.
+
+## Definitions
+
+<dl>
+<dt>Draft Release</dt>
+<dd>The release of draft document, starting a period of feedback collection from the community</dd>
+<dt>Deprecation</dt>
+<dd>Companies are no longer encouraged to use the release (e.g. to compute and subsequently publish PCFs based on it)</dd>
+<dt>Sunset</dt>
+<dd>The release is no longer supported. Companies are discouraged to use the release (e.g. to compute and to subsequently publish PCFs based on it)</dd>
+</dl>
+
+
+
+**As always, the timelines are subject to change.**
+
+## Pathfinder Framework: Upcoming Releases
+
+| When | Milestone | What |
+| --- | --- | --- |
+| December 31, 2023 | Deprecation | Pathfinder Framework v1 |
+| Mid-May to Mid-June 2024 | Draft Release | Pathfinder Framework v2.1 |
+| Q3 2024 | Published Release | Pathfinder Framework v2.1 |
+| Q3 2024 (Same date as release of updated Pathfinder Framework (v2.1)) | Deprecation | Pathfinder Framework v2.1 |
```

Comment:
Joy caught a typo in our Notion page. This line should read Pathfinder Framework v2.0

Reply:
fixed in 99faba5fe22732dabc17fe15ebd0e421f7cf377c as well